utilities/test_suite/HOST/Tensor_host.cpp

```diff
@@ -187,10 +187,10 @@ int main(int argc, char **argv)
 
     // Get number of images and image Names
     vector<string> imageNames, imageNamesSecond, imageNamesPath, imageNamesPathSecond;
-    search_jpg_files(src, imageNames, imageNamesPath);
+    search_files_recursive(src, imageNames, imageNamesPath, ".jpg");
```

Comment:
Do we need to touch the non-audio tests in this PR? (Tensor_host.cpp and Tensor_hip.cpp changes)

Reply:
We have made the recursive file search generic, so that valid extensions are passed as input, so that it can be used for both image and audio

@HazarathKumarM May be you can issue this minor change as seperate PR to abishek fork and this can be merged as part of test suite PR3 itself. Since that PR is currently open, we can may be add this change, so that this change won't show up in audio PR

Reply:
We have made changes to use same recursive file search function for both audio and image, so we are passing extensions as input to the function

Reply:
If this change has a specific reason why its in the Audio PR, that's okay.  
But if there are no dependencies for Tensor_host.cpp and Tensor_hip.cpp, would suggest putting it in the Test suite upgrade 3 PR.